README.md

```diff
@@ -88,7 +88,7 @@ Font enumeration can help by enabling:
 // Asynchronous Query and Iteration
 (async () => { // Async block
   // May prompt the user:
-  const status = await navigator.permissions.request({ name: "local-fonts" });
+  const status = await navigator.permissions.request({ name: "font-access" });
```

Comment:
For a later PR: since it's unlikely that this permissions API will be out before the font access API itself, should we remove these?

Reply:
I'm going to switch the examples to use the `permissions.query()` API. The example still stands.